src/mudlet-lua/lua/geyser/GeyserTests.lua

```diff
@@ -53,7 +53,7 @@ function Geyser.testGauges()
   end
 end
 
---- Used by Geyser.demo1(). Sorry about the namespace polution,
+--- Used by Geyser.demo1(). Sorry about the namespace pollution,
```

Comment:
I've seen both on-line and, off-hand, I am not sure which is correct.

Reply:
The bot is correct see https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=Polution

Reply:
The bot is correct see https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=Polution